### ISSUE TITLE
Login screen suggestions

### ISSUE THREAD

**Issue by towlie** [qTox/qTox#1788]
~~Make "New Profile" and "Login" look like clickable buttons.~~

~~Make those buttons including the green one change on mouseover (color change?).~~
Add a mouseover effect to the "new profile", "login" tabs and the login button.

**Comment by agilob** [comment]
- checkbox `always start this profile`
  or automatically load profile when there is only one available.

**Comment by tux3** [comment]
The autostart checkbox is definitely planned.
EDIT: It's done.

**Comment by towlie** [comment]
Ignore this comment, there is a separate issue: #1807

---

The auto login checkbox should disappear when a encrypted profile is selected or should be unticked and locked.

**Comment by linux-modder** [comment]
@towlie  on encrypted ones it can /does (for single accounts like mine)  auto launch it just holds for passphrase do however like the mention about graying it out on such profiles tho...

**Comment by ovalseven8** [comment]
Closable because hover effect is implemented.

**Comment by zetok** [comment]
> Closable because hover effect is implemented.

Not quite, since there is still no hover effect for tabs - though, I guess that I could fix it.

**Comment by zetok** [comment]
# 1

![1a](https://cloud.githubusercontent.com/assets/3148759/10561497/d6b588e4-7522-11e5-9b9b-d1450ad399fa.png)
![1b](https://cloud.githubusercontent.com/assets/3148759/10561500/0d2b1272-7523-11e5-835d-3d6a70e339cb.png)

# 

# 2

![2a](https://cloud.githubusercontent.com/assets/3148759/10561509/988fda32-7523-11e5-9d43-bc921c0f866a.png)
![2b](https://cloud.githubusercontent.com/assets/3148759/10561511/9d208af6-7523-11e5-8db6-4e54862d3f6a.png)

# 

# 3

![3a](https://cloud.githubusercontent.com/assets/3148759/10561517/d39c818e-7523-11e5-8a21-1bfcb9f53a22.png)
![3b](https://cloud.githubusercontent.com/assets/3148759/10561519/d9c1b5ca-7523-11e5-8a3d-d2eefde113d9.png)

# 

Which one?

**Comment by towlie** [comment]
2

**Comment by BiTOk** [comment]
1. 
   Everywhere in the form lines are straight

**Comment by zetok** [comment]
# 4

![4?](https://cloud.githubusercontent.com/assets/3148759/10561565/179b7c76-7526-11e5-88d3-e978d60d8107.png)
mockup - note though that I doubt whether I could make it look like that, so most likely someone else will have to do it

**Comment by BiTOk** [comment]
May be 4 without horizontal delimiter?

**Comment by zetok** [comment]
Actually, I wonder whether highlighting buttons on the left is really needed…

**Comment by ProMcTagonist** [comment]
Definitely 4. The line needs to be removed on hover, though, like @BiTOk said.

**Comment by ovalseven8** [comment]
Number 3 looks most elegant.

**Comment by linux-modder** [comment]
#3  my fav too

**Comment by DanTheBritish** [comment]
4 with no line 

someone should strawpoll this

**Comment by agilob** [comment]
The line is already there as a separator and IMO it should stay there to keep UI consistent between views on mouse hover and without mouse hover.
